spirit/comment/history/views.py

```diff
@@ -17,6 +19,12 @@ def detail(request, comment_id):
     comment = get_object_or_404(Comment.objects.for_access(request.user),
                                 pk=comment_id)
 
+    # not comment author and not moderator:
+    if request.user != comment.user and not request.user.st.is_moderator:
```

Comment:
`... and ST_PRIVATE_COMMENT_HISTORY` is missing.

Reply:
Agree.